draft-ietf-quic-multipath.md

```diff
@@ -634,6 +624,13 @@ cannot be issued more than once on the same connection
 and therefore are unique for the scope of the connection,
 regardless of the associated Path ID.
 
+{{Section 5.1.2 of QUIC-TRANSPORT}} indicates that an endpoint
+can change the connection ID it uses to another available one
+at any time during the connection. For the extension specified in
+this draft, only connection IDs associated to the same
+Path ID MUST be used on the same path. Use of a connection ID assocaited with
+a another Path ID will be considered as an attempt to open new path instead.
+
```

Comment:
"For the extension specified in
this draft, only connection IDs associated to the same
Path ID MUST be used on the same path." 

That's a bit convoluted. How about, 

For the extension specified in
this draft, endpoints MUST only migrate to another connection IDs associated to the same
Path ID.

Reply:
Not sure about the word "migrate" because you could also change the CID without connection migration...

Reply:
You are right. The correct word is "rotate".

Reply:
Or just say "use".

Reply:
I tried "use" but that doesn't work because it's about the process of changing the CID to another on the same path.  But rotate works!